frontend/src/dummyData/Contacts/columns.ts

```diff
@@ -0,0 +1,9 @@
+import { GridColDef } from "@mui/x-data-grid";
```

Comment:
at some point, we should consider moving this dummyData out of the frontend, and into the project root, and then consuming it on the frontend (as you are here) and also in the backend as seeds.

Reply:
we'll do it once we've made the endpoint in another ticket, most likely. no worries for now!